target/linux/ath79/image/generic.mk

```diff
@@ -438,6 +438,14 @@ define Device/jjplus_ja76pf2
 endef
 TARGET_DEVICES += jjplus_ja76pf2
 
+define Device/kuwfi_830d
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := KuWfi
+  DEVICE_MODEL := 830D
+  DEVICE_PACKAGES := uboot-envtools kmod-i2c-gpio
```

Comment:
You might have overlooked the part about uboot-envtools - it is also part of the target packages, so it could be dropped.

Also - we seem to have nothing ayttached via i2c. So the `DEVICE_PACKAGES` might be dropped altogether.

Reply:
fixed

Reply:
fixed

Reply:
fixed

Reply:
fixed